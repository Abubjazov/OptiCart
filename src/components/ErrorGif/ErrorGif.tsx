import errorGif from './error.gif'

export const ErrorGif = (): JSX.Element => (
	<img
		style={{
			display: 'block',
			width: '250px',
			height: '250px',
			objectFit: 'contain',
			margin: '0 auto',
		}}
		src={errorGif}
		alt='An error has occurred'
	/>
)
