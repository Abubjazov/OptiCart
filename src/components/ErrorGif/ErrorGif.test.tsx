import { render } from '@testing-library/react'
import { ErrorGif } from './ErrorGif'

test('renders ErrorGif', () => {
	const { asFragment } = render(<ErrorGif />)
	expect(asFragment()).toMatchSnapshot()
})
