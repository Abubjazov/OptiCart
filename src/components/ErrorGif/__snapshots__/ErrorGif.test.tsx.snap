// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ErrorGif should render ErrorGif 1`] = `
<DocumentFragment>
  <img
    alt="An error has occurred"
    src="error.gif"
    style="display: block; width: 250px; height: 250px; object-fit: contain; margin: 0px auto;"
  />
</DocumentFragment>
`;
